### DISCUSSION TITLE
GDScript Syntax

### DISCUSSION THREAD

**Discussion by MrAlex2** [Acode-Foundation/Acode#766] [Ideas]
GDScript is a language for Godot Engine. I work with him all the time, and it would be great to see his syntax in this application (especially versions 1 and 2), almost no one adds it to similar apps. I hope you will think about it

**Comment by legend-sabbir** [comment]
contact me in telegram . Because im not familiar with the syntaxes .

**Reply by sebastianjnuwu** [reply]
Hi, do you have email? (About astro syntax)

**Reply by legend-sabbir** [reply]
better download telegram 😕 . Also acode have official group on telegram .

**Reply by sebastianjnuwu** [reply]
I don't have it, telegram is blocked in my country :/ open a discussion

**Reply by legend-sabbir** [reply]
just use VPN man😕

**Reply by sebastianjnuwu** [reply]
To bind you need a cellular operator, and the government is not such a beast "-"

**Reply by sebastianjnuwu** [reply]
Why doesn't acode have a discord? Everyone has.

**Comment by sebastianjnuwu** [comment]
> GDScript is a language for Godot Engine. I work with him all the time, and it would be great to see his syntax in this application (especially versions 1 and 2), almost no one adds it to similar apps. I hope you will think about it

What is the file extension and icon?

**Reply by bajrangCoder** [reply]
Extension: `.gd`
Icon: [Godot](https://github.com/bajrangCoder/acode-ayu-mirage-theme/blob/main/dist/icons/godot.svg)

**Reply by sebastianjnuwu** [reply]
"-" the plugins are increasingly difficult to understand kkkk

**Reply by bajrangCoder** [reply]
Which plugin?

**Reply by sebastianjnuwu** [reply]
Why do I build my code based on other people's code why the documentation is kind of sparse last time I looked at how to implement the highlighting I wanted to put in for astro it seemed easy now not anymore

**Reply by bajrangCoder** [reply]
If you want to write highlighting rules etc, then you have to follow ace docs because Acode uses ace so here Acode docs doesn't matter, 

I know ace highlighting docs is also complex.

**Reply by legend-sabbir** [reply]
discord username: legendsabbir

**Reply by sebastianjnuwu** [reply]
I saw that I managed to change the folder icon and add the regex can you help me in a new thread?

**Reply by bajrangCoder** [reply]
Ok

**Comment by bettywolfy** [comment]
Same, I'm in need of documentation on how to create customizable syntax highlighting rules.
